lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_release.rb

```diff
@@ -11,17 +11,31 @@ def self.run(params)
         other_action.ensure_git_branch(branch: 'develop')
 
         # Create new configuration
-        @new_short_version = Fastlane::Helper::Android::VersionHelper.bump_version_release()
-        create_config()
-        show_config()
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
```

Comment:
```suggestion
        app = params[:app]
```
The fallback is already handled by fastlane given how that param is defined in `ConfigItem` below

Reply:
Ahhh Right!